src/base/ddl_log.c

```diff
@@ -386,6 +386,15 @@ logddl_set_stmt_type (int stmt_type, PT_NODE * statement)
       is_executed_ddl_for_trans = true;
       is_executed_ddl_for_csql = true;
 
+      if (stmt_type == PT_METHOD_CALL)
+	{
+	  PT_METHOD_CALL_INFO *call = &statement->info.method_call;
+	  if ((call->call_or_expr == PT_IS_CALL_STMT)
+	      && (strcasecmp (call->method_name->info.name.original, "login") == 0))
+	    {
+	      return true;
+	    }
```

Comment:
What will happen in else?

Reply:
The idea is to not increment ddl_audit_handle.ddl_stmt_cnt only when login().